src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-manager.context.ts

```diff
@@ -1,8 +1,9 @@
 import { Injectable } from '@angular/core';
 import { SearchComposite } from '../shared/model/search-composite';
+import { GermplasmListModel } from '../shared/germplasm-list/model/germplasm-list.model';
 
 @Injectable()
 export class GermplasmManagerContext {
     searchComposite: SearchComposite<any, number>;
```

Comment:
GermplasmManagerContext ideally  not be used for GermplasmListCloneComponent as it's more proper in germplasm list context/module/route